src/openloco/windows/CompanyFaceSelection.cpp

```diff
@@ -212,17 +192,17 @@ namespace openloco::ui::windows::CompanyFaceSelection
     // 0x004352C7
     static void scrollMouseOver(window* const self, const int16_t x, const int16_t y, const uint8_t scroll_index)
     {
-        const auto [rowIndex, object] = getObjectFromSelection(y);
+        auto [rowIndex, object] = getObjectFromSelection(y);
         if (self->row_hover == rowIndex)
         {
             return;
         }
         self->row_hover = rowIndex;
-        self->object = object._name;
-        sub_471B95();
+        self->object = &object;
```

Comment:
You can't do this this is a stack variable you are taking the address of. I had purposely been a bit sneaky and passed in the _name which is not a stack variable.